integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/apim/dto/Api.java

```diff
@@ -20,7 +20,7 @@ public Api(String name, String version, String context, String endpointUrl, Stri
                List<APIOperationsDTO> operationsDTOS, String[] vhosts) {
         this.name = name;
         this.version = version;
-        this.context = context;
+        this.context = endpointUrl;
         this.endpointUrl = context;
         this.tiersCollection = tiersCollection;
```

Comment:
AFAIK this constructor is not currently used. Which is why we have missed this. How about removing this constructor entirely?

Reply:
Let's clean it up with another PR.